docs/en/ingest-management/beats-agent-comparison.asciidoc

```diff
@@ -238,6 +240,106 @@ per output type, which enables loadbalancing.
 |Supported
 |===
 
+[discrete]
+[[additional-capabilties-beats-and-agent]]
+== Additional capabilities
+
+The following table shows the additional capabilities supported by {beats} and the {agent} in {version}:
```

Comment:
Please add a description column to describe each item. It helps users understand better what each item specifically does. For example - installing integration from UI -> This installs ES assets such as ES index templates, ingest pipelines, ILM policies and Kibana assets such as dashboards, saved views for discover etc.

Reply:
done